src/data/media-player.ts

```diff
@@ -340,6 +349,52 @@ export const computeMediaControls = (
   return buttons.length > 0 ? buttons : undefined;
 };
 
+export const computeMediaControls = (
```

Comment:
Instead of renaming old method, let's add an optional options param to opt-in to extended buttons.